### PR TITLE
Run testcafe in Travis

### PR DESCRIPTION
**Pull request by cdrini**
Followed instructions on https://devexpress.github.io/testcafe/documentation/continuous-integration/travis.html

- Added an `npm run serve` command that launches the app on localhost:8000
- `npm run testcafe:e2e` now starts the app itself; might conflict?

I'm a little nervous they might be flaky; but lets see.

### CODE REVIEW

File: .testcaferc.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "appCommand": "npm run serve",
```
Comment (iisa) [inline .testcaferc.json]: Q: how is this command run? I did not see it explicitly called in the `test:e2e` script

Comment (cdrini) [reply .testcaferc.json]: `test:e2e` calls `testcafe`, which looks at this config file (and runs this command) 🕸

Comment (iisa) [reply .testcaferc.json]: ah ok, thanks for clarifying. i was looking at the command line docs, but finally found this `https://devexpress.github.io/testcafe/documentation/using-testcafe/configuration-file.html#appcommand`

**Comment by iisa** [comment]
🍰

**Pull request merged**